### ISSUE TITLE
cph is generating empty metadata.json file

### ISSUE THREAD

**Issue by analog-cbarber** [conda/conda-package-handling#88]
I converted a .tar.bz2 package to a .conda package using:

```
cpy transmute old.tar.bz2 .conda
```

when I unzip the file, I see that the `metadata.json` file is empty, but based on [this code](https://github.com/conda/conda-package-handling/blob/ea093b3ac475cd19a9b11a740933b60cef0ab376/src/conda_package_handling/conda_fmt.py#L82-L91)
I expect it to contain:

```json
{"conda_pkg_format_version":2}
```

This behavior may be OS or python version specific. I am using Python 3.7.3 on MacOS.

**Comment by analog-cbarber** [comment]
If I insert a flush I get the expected behavior:

```python
           with ZipFile(conda_pkg_fn, 'w', compression=ZIP_STORED) as zf:
                with NamedTemporaryFile(mode='w', delete=False) as tf:
                    json.dump(pkg_metadata, tf)
                    tf.flush()
                    zf.write(tf.name, 'metadata.json')
```

**Comment by jezdez** [comment]
Fixed in #89.
